pkg/secrets/passdriver/passdriver_test.go

```diff
@@ -0,0 +1,85 @@
+package passdriver
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func setup() (*Driver, error) {
+	return NewDriver("gopass")
```

Comment:
I'd love to also execute `pass(1)` to make sure the driver works with it as well.

Reply:
May require to turn them into table-driven tests.

Reply:
The new implementation is agnostic about pass/gopass, so I guess this counts as resolved